README.md

```diff
@@ -15,8 +15,19 @@ priority or that stop publishing for any reason.  The stream currently in use is
 * `/any` (`geometry_msgs/msg/Twist`) - One topic of type `geometry_msgs/msg/Twist` is subscribed to based on the `topic` parameter (see Parameters below).  The data from the highest-priority of these inputs is used as the output `/cmd_vel`.
 
 ## Parameters
-* `name` (list of strings) - The "name" of each of the input `geometry_msgs/msg/Twist` topics.  This is what will be published on the `active` topic when the currently active publisher changes.
-* `topics` (list of strings) - The "topic" corresponding to each of the input `geometry_msgs/msg/Twist` topics.  This is what will be subscribed to.  The length of this list must match the `name` list above.
-* `timeout` (list of doubles) - The "timeout" of each of the input `geometry_msgs/msg/Twist` topics.  If no data is received on the input topic for this amount of time, the input will be automatically disabled.  The length of this list must match the `name` list above.
-* `priority` (list of integers) - The "priority" of each of the input `geometry_msgs/msg/Twist` topics.  The higher the number, the higher the priority.  Higher priority topics will dislodge lower priority topics and start publishing to the output topic automatically.  The length of this list must match the `name` list above.
-* `short_desc` (list of strings) - The "short description" of each of the input `geometry_msgs/msg/Twist` topics.  This is informational only.  The length of this list must match the `name` list above.
+Subscribers use a dictionary way of being set up, with `name` being the key:
+* `name` (string) - The "name" of each of the input `geometry_msgs/msg/Twist` topics.  This is what will be published on the `active` topic when the currently active publisher changes.
+* `topic` (string) - The "topic" corresponding to each of the input `geometry_msgs/msg/Twist` topics.  This is what will be subscribed to.  The length of this list must match the `name` list above.
+* `timeout` (double) - The "timeout" of each of the input `geometry_msgs/msg/Twist` topics.  If no data is received on the input topic for this amount of time, the input will be automatically disabled.  The length of this list must match the `name` list above.
+* `priority` (integer) - The "priority" of each of the input `geometry_msgs/msg/Twist` topics.  The higher the number, the higher the priority.  Higher priority topics will dislodge lower priority topics and start publishing to the output topic automatically.  The length of this list must match the `name` list above.
+* `short_desc` (string) - The "short description" of each of the input `geometry_msgs/msg/Twist` topics.  This is informational only.  The length of this list must match the `name` list above.
```

Comment:
Forgot to delete 'The length of this list must match the `name` list above.' in the description of the elements of the disctionary, sorry

Reply:
No worries.  Feel free to open other PRs against this one, I'm happy to merge.